setup.py

```diff
@@ -67,13 +67,13 @@
             "pytest-cov>=3.0.0",
             "pytest-notebook>=0.6.1",
             "pytest-xdist>=2.4.0",
+            "opencv-contrib-python-headless==4.1.2.30",
             "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
             "tensorflow>=2.4.0",  # For rllib tests
         ],
         "train": [
-            "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
-            # XXX: TF requires specific version of scipy
-            "scipy==1.4.1",
+            "opencv-contrib-python-headless==4.1.2.30",
```

Comment:
why was `scipy` removed?

Reply:
`scipy` did not appear to be needed. I think I will return it for now.